_posts/2017-06-27-rethinking-geocoders-adding-local-vernacular-into-the-build-process.md

```diff
@@ -13,25 +13,25 @@ tags:
+To remedy the unnecessary need for prefix directions, we programmatically analyzed every address point and road centerline in the State checking if it could geocode without a prefix. If the address point or road centerline is unique, and can be located without a prefix direction, it is then placed in a validated table that is utilized during the geocoding process. The end result is that you now have the ability to geocode verified addresses with or without a prefix direction. For example you will now get a 100% match score when geocoding 3305 Wasatch Drive or 3305 S Wasatch Drive. Likewise for 1575 State Street and 1575 S State Street.
 
-To improve performance and search time, we programmatically created a derived road centerline feature class.  This is essentially a denormalized version of the SGID roads feature class.  Diagram 3 illustrates this well, showing how the SGID road segment in Diagram 2 gets denormalized into four separate features.  This feature class is specifically created for geocoders and is recreated monthly before we rebuild the road centerline geocoders.  The idea behind this derived feature class is that we now create one road centerline geocoder that takes into account the various alias names for each segment.  This eliminates the need for us to build four separate geocoders and a composite geocoder.  This equates to faster, more efficient geocoding.
+To improve performance and reduce the number of necessary locators, we exploded the road centerline feature class. This is a denormalized version of the SGID roads feature class. Diagram 3 illustrates this well, showing how the SGID road segment in Diagram 2 gets denormalized into four separate features. This feature class is specifically created for geocoders and is recreated monthly before we rebuild the road centerline geocoders. The idea behind this derived feature class is that we now create one road centerline geocoder that takes into account the various alias names for each segment. This eliminates the need for us to build and maintain four separate geocoders and a composite geocoder. This equates to faster, more efficient geocoding.
 
-When implemented (TBD), we not only expect our geocoding services to be more versatile and quicker, but we expect better results (better match scores) and happier end users.  We look forward to hearing your feedback. 
+When implemented, we expect our geocoding services to be more versatile, quicker, with higher match scores, and happier end users. We look forward to hearing your feedback. 
```

Comment:
This sentence is still a bit awkward. I like the first version better expect "quicker" sounds better as "quick" to me. Maybe "performant" would also work.